go/private/repositories.bzl

```diff
@@ -103,6 +104,9 @@ def go_rules_dependencies():
       name = "gogo_special_proto",
   )
 
+  _maybe(googleapis_repository,
```

Comment:
It doesn't seem like this needs to be a new repository rule. Can you use `git_repository` with `overlay` instead?